components/brave_new_tab_ui/components/default/settings/index.ts

```diff
@@ -627,8 +628,11 @@ export const StyledSelectionBorder = styled('div')<SelectionProps>`
       right: 10px;
     }
   `}
-`
 
+  ${p => p.removable && css`
+    &:hover:after { display: none; }
```

Comment:
Optional nit: So both `:after` and `::after` do the same thing, but for semantic reasons `::after` is preferred:

`::` for pseudo elements (e.g. `::after`, `::before`, `::first-letter`)
`:` for pseudo classes (e.g. `:focus`, `:hover` ect)

https://stackoverflow.com/a/48182482/3260044